### DISCUSSION TITLE
Gmail und IMAP?

### DISCUSSION THREAD

**Discussion by JPRuehmann** [tine-groupware/tine#77] [Q&A] [answered]
Hallo
Wird es irgendwann eine Version von Tine geben die mit Gmail zusammen funktioniert?
In der alten reichte es ein APP paswort in seinem Google Account anzulegen.
Das funktionierrt jetzt aber nur noch für smtp.
Mit imap wird das APP Passwort nicht akzeptiert.
Danke.

**Comment by pschuele** [comment, marked as answer]
siehe https://github.com/tine-groupware/tine/issues/75 - sollte eigentlich funktionieren.

**Comment by JPRuehmann** [comment]
Imap geht nicht mehr mit APP-Passwort nur noch SMTP
Das wird aber nicht ewig gehen.
Auf die dauer wird es nur noch 0Auth geben.
